tests/acceptance/expected-failures-webUI-on-OCIS-storage.md

```diff
@@ -1,14 +1,12 @@
-## Scenarios from web tests that are expected to fail on OCIS with OCIS storage
+## Scenarios from web tests that are expected to fail on OCIS with owncloud storage
```

Comment:
```suggestion
## Scenarios from web tests that are expected to fail on OCIS with OCIS storage
```

This error is in the expected-failures file in web also - we can fix it separately.

Reply:
yes please :-)